jwst/extract_2d/tests/test_grisms.py

```diff
@@ -148,7 +148,7 @@ def create_tso_wcsimage(filtername="F277W", subarray=False):
     im.meta.wcsinfo.siaf_xref_sci = 887.0
     im.meta.wcsinfo.siaf_yref_sci = 35.0
     aswcs = AssignWcsStep()
-    return aswcs.process(im)
+    return aswcs.process(im, sip_approx=False)
```

Comment:
Hm, I'm surprised you have to disable sip approximation in this tests. The test is for NRC_WFSS (grism) observations and normally should not go through sip approximation.

Reply:
oh whoops, i was just blindly going through and switching it off every time AssignWcsStep was called in the tests. ill fix that.